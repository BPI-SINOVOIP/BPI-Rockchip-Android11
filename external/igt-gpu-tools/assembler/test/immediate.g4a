mov (1) g0<1>UD 4294967295UD { align1 };
mov (1) g0<1>UD 2147483647D { align1 };
mov (1) g0<1>UD -2147483648D { align1 };
