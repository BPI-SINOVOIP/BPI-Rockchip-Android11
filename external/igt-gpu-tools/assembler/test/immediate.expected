   { 0x00000001, 0x20000061, 0x00000000, 0xffffffff },
   { 0x00000001, 0x200000e1, 0x00000000, 0x7fffffff },
   { 0x00000001, 0x200000e1, 0x00000000, 0x80000000 },
